src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptyforinitializerpad/InputEmptyForInitializerPad2.java

```diff
@@ -7,5 +7,5 @@
 
 package com.puppycrawl.tools.checkstyle.checks.whitespace.emptyforinitializerpad;
 
-class InputEmptyForInitializerPad2 // violation
+class InputEmptyForInitializerPad2 // violation 'Invalid exception message'
```

Comment:
How did we get this message for this check?

https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages
I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

Reply:
@suniti0804 Also, there is a conflict.

Reply:
> How did we get this message for this check?
> 
> https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

actually, there is no any violation for this input class, so i removed the // violation comment, same has been found for an input file using RegexpCheckTest as well - https://github.com/checkstyle/checkstyle/pull/13965#discussion_r1375303142

Reply:
> How did we get this message for this check?
> 
> https://checkstyle.org/checks/whitespace/emptyforinitializerpad.html#Violation_Messages I am only seeing `is preceded with whitespace` or `is not preceded with whitespace`.

acually, in this input class there is no any violation of the kind specified, so in my latest commit i removed the // violation comment. same has been found for an input file using RegexpCheckTest - https://github.com/checkstyle/checkstyle/pull/13965#discussion_r1375303142